crates/router/src/configs/defaults.rs

```diff
@@ -136,3 +142,300 @@ impl Default for super::settings::DrainerSettings {
         }
     }
 }
+
+// Test Data
```

Comment:
..

Reply:
I have told connector team to fill this data once my PR gets merged